DOL.WHD.Section14c.Web/src/modules/pages/employerRegistrationTemplate.html

```diff
@@ -0,0 +1,122 @@
+<dol-header></dol-header>
+<main-top-nav-control></main-top-nav-control>
+<div ng-show="registrationSuccess">
+  <div  class="dol-feedback-success">
+    <p class="dol-feedback-message">
+      <strong>You Have Created an Employer</strong>
```

Comment:
>Brava!

I feel like some confetti or something would be appropriate, too.  😜

Sorry, I'm a little snarky from an early morning mostly spent on an airplane.  I just thought this very blunt message was kind of funny.

Reply:
haha no worries